.meteor/packages

```diff
@@ -31,5 +31,8 @@ space:testing-event-sourcing
 
 # testing
 test-in-console
+# TODO: This fixes tests after upgrading to Meteor 1.3.4.1, thins needs further investigation
+practicalmeteor:mocha-core@0.1.4
```

Comment:
`practicalmeteor:mocha-core`is pegged at `0.1.4` since test with version `1.0.0` and `1.0.1` fail with:

```javascript
Error: Match error: Failed Match.OneOf, Match.Maybe or Match.Optional validation in field targetId
    at exports.check (packages/check/match.js:34:1)
    at OpenBankAccount._checkFields (packages/space:testing-event-sourcing/source/allow-to-ignore-some-struct-types.js:17:5)
    at OpenBankAccount.Struct (packages/space_base/source/struct.coffee:7:6)
    at OpenBankAccount.Constructor (packages/space:messaging/source/serializables/command.js:11:25)
    at OpenBankAccount.Command (eval at <anonymous> (packages/space_base.js:312:13), <anonymous>:2:40)
    at OpenBankAccount.Constructor (packages/space:domain/source/command.js:7:36)
    at OpenBankAccount.Command (eval at <anonymous> (packages/space_base.js:312:13), <anonymous>:2:40)
    at OpenBankAccount.Constructor (packages/space_base.js:310:21)
    at new OpenBankAccount (eval at <anonymous> (packages/space_base.js:312:13), <anonymous>:2:48)
    at Test.<anonymous> (app-tests/server/bank-account.app-tests.js:41:11)
```
@DominikGuzei: can you take a look at this when you have time? We can not use current versions of `practicalmeteor:mocha` because of this.